src/components/Tool/NavToolList/ToolNavItems.ts

```diff
@@ -0,0 +1,46 @@
+export const ToolNavItems = [
```

Comment:
Could you combine/share this and [`ToolsList`](https://github.com/CruGlobal/mpdx-react/pull/132/files#diff-e25bacfc25a4c1c9b4558c1d31a1b232268dbc405f7a522e56f0f130095b5017R15) somehow?